ui_framework/components/pager/pager.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { KuiPagerButtons } from './pager_buttons';
+
+export function KuiPager({
+    className,
+    startNumber,
+    endNumber,
+    totalItems,
+    hasPreviousPage,
+    hasNextPage,
+    onNextPage,
+    onPreviousPage,
+    ...rest }) {
+  const classes = classNames('kuiPager', className);
+  return (
+    <div className={classes} { ...rest }>
+      <div className="kuiPagerText">{startNumber}&ndash;{endNumber} of {totalItems}</div>
+      {
+        startNumber === 1 && endNumber === totalItems
```

Comment:
For anyone not quite sure about the order of operations when it comes to evaluating ternaries, how about wrapping this condition in parens?

```javascript
(startNumber === 1 && endNumber === totalItems)
```